pkg/gateway/api/api_grpc_test.go

```diff
@@ -2236,6 +2240,272 @@ func TestGetTargetFeatures(t *testing.T) {
 	}
 }
 
+var allFeaturesForPrerequisiteTest = map[string]*featureproto.Feature{
```

Comment:
This var is same as this figure.

![image](https://user-images.githubusercontent.com/2755429/231992929-4135c8b3-2f27-4977-b4c2-1cd3daa35b33.png)
https://github.com/bucketeer-io/bucketeer/pull/283/files#diff-4f9bf2864f5558559c0c7538472e2d28df9d965beee1d46db1315aef6231ae6dR53-R65